woocommerce/woocommerce
# Add CSS outline for keyboard accessibility

### Changes proposed in this Pull Request:

Closes #50787.

Adds outline and applies hover effects on site visibility badge for keyboard accessibility.

https://github.com/user-attachments/assets/33c0851c-589b-4a33-9e62-0cd57c320572

### How to test the changes in this Pull Request:

1. Go to `/wp-admin`
2. Click on an empty part of the admin bar to reset the focus index
3. Press on tab a few times and observe the site visibility badge is outlined when it's focused
4. Go to `WooCommerce > Settings > Site visibility`
5. Change to either `Live` or `Coming soon`
6. Test the badge again on step 3 and observe the similar focus effect is applied
7. Hover the badge and see that no outline is shown as the outline is intended for focus state only



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
